microsoft-edge/devtools/network/reference.md

```diff
@@ -734,97 +1372,220 @@ To see both the transferred size and the uncompressed size of a resource at the
+
+   Or, in the action bar at the top, click the **Export HAR (either sanitized or with sensitive data)** (![Export HAR icon](./reference-images/export-har-sanitized-icon.png)) button, and then select **Export HAR (with sensitive data)**:
+
+   ![Selecting 'Copy all as HAR (with sensitive data)'](./reference-images/export-har-button.png)
 
 
 <!-- ------------------------------ -->
-#### Copy one or more requests to the clipboard
+#### Import a HAR file into DevTools for analysis
+<!-- bottom of https://developer.chrome.com/docs/devtools/network/reference/#save-as-har -->
+
+After you have a HAR file, you can import it back into DevTools for analysis by using [HAR Analyzer](https://toolbox.googleapps.com/apps/har_analyzer/).
+
+To import a HAR file into DevTools for analysis:
 
-Under the **Name** column of the Requests table, right-click a request, click **Copy**, and then select one of the following options:
+* In the action bar at the top of the **Network** tool, click the **Import HAR file** (![Import HAR file icon](./reference-images/import-har-file-icon.png)) button.
+
+* Or, drag the HAR file onto the **Requests** table.
+
+The **Network** tool reads and shows initiators for the requests that are imported from the HAR file.  For initiators, see [Display a log of requests](#display-a-log-of-requests), above.
+
+
+<!-- ------------------------------ -->
+#### Copy network requests to the clipboard
+<!-- Copy a request, a filtered set of requests, or all of them to the clipboard  https://developer.chrome.com/docs/devtools/network/reference/#copy -->
+
+Under the **Name** column of the **Requests** table, right-click a request, hover over **Copy**, and then select one of the following options.
+
+To copy a single request, its response, or <!-- todo: its? -->stack trace:
 
 | Name | Details |
 | --- | --- |
-| **Copy Link Address** | Copy the URL of the request to the clipboard. |
-| **Copy Response** | Copy the response body to the clipboard. |
-| **Copy as Fetch** | &nbsp; |
-| **Copy as cURL** | Copy the request as a cURL command. |
-| **Copy All as Fetch** | &nbsp; |
-| **Copy All as cURL** | Copy all requests as a chain of cURL commands. |
-| **Copy All as HAR** | Copy all requests as HAR data. |
+| **Copy URL** | Copy the URL of the request to the clipboard. |
+| **Copy as cURL (cmd)** | Copy the request as a cURL command. |
+| **Copy as cURL (bash)** |  |
+| **Copy as PowerShell** | Copy the request as a PowerShell command. |
+| **Copy as fetch** | Copy the request as a fetch call. |
+| **Copy as fetch (Node.js)** | Copy the request as a Node.js fetch call. |
+| **Copy response** | Copy the response body to the clipboard. |
+| **Copy stack trace** | Copy the request's stack track to the clipboard. |
+<!-- todo: "Copy stack trace" is listed in upstream, but is not in any screenshots, not seeing in UI --> 
```

Comment:
```suggestion
To copy a single request, its response, or its stack trace:
```
how get the **Copy stack trace** command?
@captainbrosset 

GH preview:
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/network-filter/microsoft-edge/devtools/network/reference.md#copy-network-requests-to-the-clipboard